sycl/test-e2e/bindless_images/sampled_fetch/fetch_3D.cpp

```diff
@@ -1,8 +1,7 @@
-// REQUIRES: cuda
-// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_3d
+// REQUIRES: aspect-ext_oneapi_bindless_images
 
```

Comment:
Same question as above, but for `// REQUIRES: aspect-ext_oneapi_bindless_sampled_image_fetch_3d`.

Reply:
Aspect has been readded and enabled in the L0 backend. Resolving.